zepto-dnd.js

```diff
@@ -661,6 +730,9 @@
   }
 
   Sortable.prototype.over = function(e) {
+    if (this.scroller)
+      this.scroller.scrollEvent(this.el, e);
+
```

Comment:
Hmm. Yes. You're right.  It's working now.

I'm not sure why it wasn't working before and had to add those lines.